sky/cli.py

```diff
@@ -1308,6 +1329,7 @@ def launch(
     use_spot: Optional[bool],
     image_id: Optional[str],
     env: List[Tuple[str, str]],
+    env_file: Optional[Dict[str, str]],
```

Comment:
```suggestion
    env_file: Optional[Dict[str, str]],
    env: List[Tuple[str, str]],
```
nit: Would be nice to keep the order in `_TASK_OPTIONS` and here to be the same.

Reply:
done~